src/main/java/com/epam/edumanagementsystem/admin/rest/api/AcademicYearController.java

```diff
@@ -84,6 +84,9 @@ private String validateStartAndEndDates(BindingResult result, Model model, Local
                 if (endDate.getMonth().maxLength() + endDate.getDayOfMonth() - startDate.getDayOfMonth() < 30) {
                     model.addAttribute("min", minRangeMsg);
                     return "academicYearSection";
+                } else if (endDate.isBefore(startDate.plusDays(30))) {
```

Comment:
performs the same action in both conditions. in my opinion can be written in one condition․                                             if ((endDate.getMonth().maxLength() + endDate.getDayOfMonth() - startDate.getDayOfMonth() < 30) || (endDate.isBefore(startDate.plusDays(30))){
}